app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,160 @@
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,700;0,900;1,300;1,400&family=Lilita+One&family=Lobster&display=swap" rel="stylesheet">
+
+<style>
+body {
+      margin: 0;
+      padding: 0;
+    }
+.navbar {
+      background-color: #007bff;
+      font-family: "Lato", sans-serif;
+      color: #fff;
+      height: 60px;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 0 20px;
+    }
+
+    p a{
+      color: #fff4;
+      font-family: "Lato", sans-serif;
+    }
+
+    .navbar h1 {
+      font-size: 18px;
+      margin: 0;
+    }
+
+    .navbar a {
+      color: #fff;
+      text-decoration: none;
+      display: flex;
+      align-items: center;
+    }
+
+    .navbar a img {
+      width: 20px;
+      height: 20px;
+      margin-right: 5px;
```

Comment:
- [ Required ] I think it would be nice if you factor out the CSS code of all HTML files in this project to a `CSS Stylesheet` and then import the stylesheet so that the styles may be applied to your pages. Currently, most of the styling of your application is done inside HTML which is not good practice. Please rectify this for the following files `views/users/index.html.erb`,  `views/categories/index.html.erb`, `views/categories/new.html.erb`, `views/expenses/index.html.erb` and `views/expenses/new.html.erb`

Reply:
@Felix45 Hi! Sorry to bother you, but I have one question related. I did all the refactor to not have that embebed code, but when I do the precompile all the CSS change and I don't have control over it, so I can not fix the linters errors. What should I do?